presets/imcs/cluster/CMakePresets.json

```diff
@@ -11,7 +11,19 @@
         "FOUR_C_ENABLE_NATIVE_OPTIMIZATIONS": "ON",
         "FOUR_C_WITH_ARBORX": "ON",
         "FOUR_C_WITH_FFTW": "ON",
-        "FOUR_C_WITH_GOOGLETEST": "OFF"
+        "FOUR_C_WITH_GOOGLETEST": "OFF",
+        "FOUR_C_TRILINOS_ROOT": "/opt/trilinos/2025-03/",
+        "MPI_C_COMPILER": "/opt/openmpi/4.1.8/gcc/bin/mpicc",
+        "MPI_CXX_COMPILER": "/opt/openmpi/4.1.8/gcc/bin/mpicxx",
+        "MPIEXEC_EXECUTABLE": "/opt/openmpi/4.1.8/gcc/bin/orterun",
+        "FOUR_C_QHULL_ROOT": "/opt/spack/v0.23.1/opt/spack/linux-ubuntu24.04-sapphirerapids/gcc-13.3.0/qhull-2012.1-52nebni444m47tb7wxonn23bbbvlcapb",
+        "QHULL_INCLUDE_DIR": "/opt/spack/v0.23.1/opt/spack/linux-ubuntu24.04-sapphirerapids/gcc-13.3.0/qhull-2012.1-52nebni444m47tb7wxonn23bbbvlcapb/include",
+        "QHULL_LIBRARY": "/opt/spack/v0.23.1/opt/spack/linux-ubuntu24.04-sapphirerapids/gcc-13.3.0/qhull-2012.1-52nebni444m47tb7wxonn23bbbvlcapb/lib/libqhull.so"
```

Comment:
If this is truly required, you either have a bug in your library setup, or 4C's configure_Qhull is not working as it should. Could you try removing these two lines and only specifying the root?